index.bs

```diff
@@ -3754,6 +3781,43 @@ extension for transaction authorization.
     });
 </pre>
 
+## Aborting Authentication Operations ## {#sample-aborting}
+
+The below example shows how a developer may use the AbortSignal parameter to abort an
+ongoing credential registration operation. A similiar procedure applies to an ongoing
+authentication operation. 
+
+<pre class="example" highlight="js">
+    const authAbortController = new AbortController(); 
+    const authAbortSignal = authAbortController.signal;
+
+    authAbortSignal.onabort = function () {
+        // Migrate away from the page and other necessary clean-ups. 
+    }
+
+    var options = {
+        // A list of options. 
+    }
+
+    navigator.credentials.create({
+        publicKey: options, 
+        signal: authAbortSignal})
```

Comment:
`signal` is a part of options yes?
so this would be..
```
     navigator.credentials.create({
         publicKey: options
     })
```
..?

Reply:
the current design is that the signal can be a separate field (as described in the example and in my other comment).

Reply:
yeah, doh, nevermind, sorry.